docs/sources/alerting/unified-alerting/high-availability/enable-alerting-ha.md

```diff
@@ -0,0 +1,32 @@
++++
+title = " Enable alerting high availability"
+description = "Enable alerting high availability"
+keywords = ["grafana", "alerting", "tutorials", "ha", "high availability"]
+weight = 450
++++
+
+# Enable alerting high availability
+
+You can [enable alerting high availability]({{< relref "./_index.md" >}}) support by updating the Grafana configuration file. On Kubernetes, you can enable alerting high availability by updating the Kubernetes container definition.
+
+## Update Grafana configuration file
+
+Gossiping of notifications and silences uses both TCP and UDP port `9094`. Therefore, each Grafana instance will need to be able to accept incoming connections on these ports.
+
+To enable high availability support:
+
+1. In your custom configuration file ($WORKING_DIR/conf/custom.ini), go to the `[unified_alerting]` section.
+2. Set `[ha_peers]` to the number of hosts for each Grafana instance in the cluster (using a format of host:port), for example, `ha_peers=10.0.0.5:9094,10.0.0.6:9094,10.0.0.7:9094`. You must have at least one (1) Grafana instance added to the [`[ha_peer]` section.
```

Comment:
The "You must" sentence should be on a separate line as it's not part of the step directive.

Reply:
Done